ungeld.lua

```diff
@@ -0,0 +1,70 @@
+-- ungelds animals
```

Comment:
This script seems to duplicate `geld.lua` quite a bit - I would suggest using `reqscript` here to import geld and run it with the ungeld option. It will be easier to maintain the scripts that way. You'll need to add module support to geld as described here for it to work: https://docs.dfhack.org/en/stable/docs/Lua%20API.html?highlight=module#scripts

Reply:
Good idea. I was thinking about converting them into a single plugin, but this would be less work for sure.

Reply:
Refactoring geld to support this looked a bit complex, so I just made ungeld use `dfhack.run_script()` instead.